### PR TITLE
Evil commit, bad

### PR DESCRIPTION
**Pull request by Ruyzambrano**

### CODE REVIEW

File: evilfile.py [hunk 1]
```diff
@@ -0,0 +1 @@
+asdkgjahslgkjhadsfkjhsdkljsldkjfhslkjdfhlkjadfhlkjsdhflkjshdfkjlhsdfalkjhsdfkjlhsdflkjhsdflkjhsd
```
Comment (Ruyzambrano) [inline evilfile.py:1]: This is BAD

**Comment by AlexOBarnes** [comment]
No this is gibberish

**Comment by Ruyzambrano** [comment]
I hate this, don't do it

**Comment by Ruyzambrano** [comment]
Opening again because

**Comment by Ruyzambrano** [comment]
dakjsdhfakjld

**Pull request closed without merging**